parts/linux/cloud-init/artifacts/components.json

```diff
@@ -355,32 +355,119 @@
       ]
     }
   ],
-  "DownloadFiles": [
+  "Packages": [
+    {
+      "name": "runc",
+      "downloadLocation": "/opt/runc/downloads",
```

Comment:
There are other scripts running at provisioning time to post-process the packages.
For example, if the files are decompressed, they will be moved to /opt/xxx/bin (e.g., /opt/crictl/bin). And if the files are in a tar, they will be untar at provisioning time.
We can certainly revise them, knowing that it will require additional effort testing both the vhd build time and provisioning time behaviors.